### ISSUE TITLE
Create view of the hierarchy in the corner of the page

### ISSUE THREAD

**Issue by lukemassa** [lukemassa/identity-mapping#2]
Basically we want a little view in the corner that shows the tree but instead of circles with words on them that move around, it is a static tree that shows the structure of the graph (think a less-ugly version of https://math.dartmouth.edu/archive/m38s02/public_html/tree.gif) 

1. Decide whether this should be in its own SVG or a different one
     a. It might be easier to position it with HTML than SVG, but it may be harder to have them interact with each other
2. Should it be contained within the Graph() object, or a different object that is updated by the Graphs
3. Have it so, during the execution of the update() function, the tree view is updated to reflect the structure of the tree

**Comment by Kakya** [comment]
So having found where most of the logic is. From what I seem to understand from Arienne and you right now, would something like the trees that exist currently in ariennej12.github.io/Map be what is desired? I can pull the code from there and merge it with the current graph layout to make it so that instead of it updating when a user presses a bootbox button, it updates when the user enters a new node in the circle graph. How does that sound? Or were you thinking of something else?

**Comment by lukemassa** [comment]
Yeah, I coded it all pretty quickly -- it could use a refactor in which I separate the logic properly into some .js files. All in all, though, I'm mostly concerned with implementing features now, not making a very robust and scalable piece of software, since this is mostly a prototype, and my main goal is to get it to a point where Arienne can show it off and get some excitement around it, then put in the time (and ideally actual developers, which I am not) to make it right!

But yeah, what you are describing sounds exactly right. One difference would be you wouldn't need the name of the node, just a little image, that way you can make it very small -- the point of this graphic is to have a sense of the whole tree (eventually there will be ways of zooming in and out of the map such that you might not see the whole tree at one time), not necessarily what the nodes are. You could maybe make them little shapes, circles or triangles or something, then copy the coloring order from the colors array in the js on the index.php page?

I'd say for now see if you can fit reasonably sized trees (<4 generations, <15 leaf nodes) in a 200x200 pixel space
